package-lock.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
Does adding this file mean there are extra install steps now? Could you add them to the Dockerfile?

Reply:
This is just for front end development purposes, if you want to (re)build the [ jsoneditor widget yourself](https://github.com/yansenlei/VJsoneditor), e.g. with some debugging (as I did) you'll want to be able to fetch all the right versions of deps/modules that are used in the build of the dist minified versions.